vaadinfrontend/src/main/java/life/qbic/datamanager/views/project/overview/components/ProjectOverviewComponent.java

```diff
@@ -307,13 +316,12 @@ private void setUpPrincipalInvestigatorSearch() {
     }
 
     private void setUpExperimentalDesignSearch() {
-      //Todo Replace with Call to service parsing information from db and generating ItemLabels
-      projectInformationDialog.analyteBox.setItems("Analyte1", "Analyte2", "Analyte3", "Analyte4",
-          "Analyte5");
-      projectInformationDialog.specimenBox.setItems("Specimen1", "Specimen2", "Specimen3",
-          "Specimen4", "Specimen5");
-      projectInformationDialog.organismBox.setItems("Organism1", "Organism2", "Organism3",
-          "Organism4", "Organism5");
+      projectInformationDialog.organismBox.setItems(
+          experimentalDesignSearchService.retrieveOrganisms());
+      projectInformationDialog.specimenBox.setItems(
+          experimentalDesignSearchService.retrieveSpecimens());
+      projectInformationDialog.analyteBox.setItems(
+          experimentalDesignSearchService.retrieveAnalytes());
```

Comment:
you could use the lazy data provider here as well, this would propagate changes in the data source to the view. As we do not expect the vocabulary to change, this is sufficient though.